src/components/Sections/SectionChart/SectionLineChart.js

```diff
@@ -47,6 +47,21 @@ const createXAxisProps = (data, dataKey, width) => {
   return props;
 };
 
+const getTimeFrame = (preparedData, chartProperties) => {
+  if (chartProperties.timeFrame !== SUPPORTED_TIME_FRAMES.none) {
+    const allNamesAreUnformattable = preparedData.every((mainGroup) => {
```

Comment:
isn't some enough ?

Reply:
just to be sure.
there are too may ways to generate a report and too many variation to send name.
I prefer to keep the original behaviour as close as I can